dispatch.rb

```diff
@@ -90,9 +94,130 @@ def send_event(name, params = {})
   end
 end
 
+class Watcher
+  JSON_SERVICES = [
+    ['http://npm-update-stream.libraries.io/', 'NPM'],
+    ['https://rubygems.org/api/v1/activity/just_updated.json', 'Rubygems'],
+    ['https://rubygems.org/api/v1/activity/latest.json', 'Rubygems'],
+    ['https://atom.io/api/packages?page=1&sort=created_at&direction=desc', 'Atom'],
+    ['https://atom.io/api/packages?page=1&sort=updated_at&direction=desc', 'Atom'],
+    ['http://package.elm-lang.org/new-packages', 'Elm'],
+    ['https://crates.io/summary', 'Cargo'],
+    ['http://api.metacpan.org/v0/release/_search?q=status:latest&fields=distribution&sort=date:desc&size=100', 'CPAN'],
+    #['https://hex.pm/api/packages?sort=inserted_at', 'Hex'],
+    #['https://hex.pm/api/packages?sort=updated_at', 'Hex']
```

Comment:
Yeah, weird things were happening on these endpoints. Assuming I should uncomment?